src/movement/thruster_sabertooth.cpp

```diff
@@ -84,12 +89,39 @@ void callBack(const robosub::thruster::ConstPtr& msg)
 
 }
 
+void setTimeOut (uint8_t ms_100)
```

Comment:
One possible thought to avoid confusion is to make this function accept a float or double in seconds, then convert it to the necessary value in uint8_t.